### ISSUE TITLE
POE+ Fan speed under OS Version 5.0.1

### ISSUE THREAD

**Issue by ChasingTraction** [FalconChristmas/fpp#1061]
I am using the official PI POE+ hat on Pi4b's (https://www.raspberrypi.org/products/poe-plus-hat/) the fan was at a low level while running os version 4.0 and 4.6. I Upgraded to 5.0 / fpp 5.0.1 and fan speed now stays at 100%

**Comment by dkulp** [comment]
I'm not able to reproduce.   I installed one on my Pi4 that I use for development and the only time I was able to get the fan to even turn on at all is when doing a "make -j 4" with an empty cache which really exercises all four the CPU core's.

**Comment by patdelaney** [comment]
I have ordered the hardware to test this. It should be here in a week or so.

**Comment by patdelaney** [comment]
I moved the POE hat Pi 4 to a quieter room and started the test. I opened a ssh sessions to the PI.  I ran the following command:
watch  cat /sys/class/thermal/cooling
It should output that value every 2 seconds to the ssh window.
I normally get a 3 but sometimes a 2
If I change the branch and compile the output jumps to 4 and the fan speeds up
When the compile is done, it drops down to 3.
I don't seem to have the issue reported.
